crates/bevy_gizmos/src/gizmos.rs

```diff
@@ -464,7 +502,9 @@ impl<'s> Gizmos<'s> {
         }
     }
 
-    /// Draw an arc, which is a part of the circumference of a circle.
+    /// Draw an arc, which is a part of the circumference of a circle, in 2D, for the current frame.
+    ///
+    /// This should be called for each frame the arc needs to be rendered.
     ///
```

Comment:
I'm sure it's wrong, and I think you're kinda right because it's using `sin_cos` and passing it in a `Vec2` for the linestrip (so x=sin and y=cos, which makes the angle relative to Y instead of X and the angle clockwise). I didn't test it though. And maybe it could be changed, that feels kinda weird.

Changed to (if we just mention the middle point it should technically be from the origin of the world):

```rust
    /// - `direction_angle` sets the clockwise  angle in radians between `Vec2::Y` and
    /// the vector from `position` to the midpoint of the arc.
```